### PR TITLE
pract-78-create a deck display card, pract-70-update homepage styling

### PR DESCRIPTION
**Pull request by elenademi**
## Description

This PR creates and formats a deck display card (Jira ticket 78) and updates styling of the home page based on the Figma prototype.

## Acceptance Criteria

The styling should use the UI kit and follow the Figma wireframes.
The deck card should display the title of the deck, the topic, and the number of cards.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

Screenshot 2023-12-10 200540

© 2022 GitHub, Inc.
Terms
Privacy
Security
Status

### CODE REVIEW

File: src/components/deckCard/DeckCard.css
Comment (nikkigraybeal) [inline src/components/deckCard/DeckCard.css:1]: - add `margin: 5px;` to all elements rendered inside of .deck-card
- hardcode .h3-deck-card with `font-size: 20px;`
- replace `width: 40%` with `width: 300px;` in .deck-card
- remove the media query from this file

File: src/pages/home/Home.css
Comment (nikkigraybeal) [inline src/pages/home/Home.css:1]: - add `width: 100%` and `margin: 0 auto` to .decks-container
- add this media query to this file: 
```
@media (max-width: 750px) {
  .decks-container {
     justify-content: center;
  }
}
```

**Review by nikkigraybeal — CHANGES_REQUESTED** [review]
The Deck card is looking really nice, Elena! Great work! 
There is margin being automatically applied to the content of the card and is making it too tall, I think. Also, we can use some adjustment on width. See comments for each file below.

**Comment by elenademi** [comment]
Thanks a lot for the feedback! I had to work all day today, but I'll get to
it tomorrow.

**Review by madebymiguel — APPROVED** [review]
LGMT

**Pull request merged**